rectools/model_selection/cross_validate.py

```diff
@@ -120,19 +121,23 @@ def cross_validate(  # pylint: disable=too-many-locals
         test_users = interactions_df_test[Columns.User].unique()
         prev_interactions = fold_dataset.get_raw_interactions()
         catalog = prev_interactions[Columns.Item].unique()
-
+        test_fold_context = get_context(interactions_df_test)
```

Comment:
```suggestion
        test_fold_context = None
        if any([model. require_recommend_context for model in models]):
            test_fold_context = get_context(interactions_df_test)
```
To remove unnecessary computations when they are not needed

Reply:
`if any(model. require_recommend_context for model in models):`

Reply:
fix